GeosCore/fullchem_mod.F90

```diff
@@ -204,9 +203,13 @@ SUBROUTINE Do_FullChem( Input_Opt,  State_Chm, State_Diag,                 &
 
     ! OH reactivity and KPP reaction rate diagnostics
     REAL(fp)               :: OHreact
-    REAL(dp)               :: Vloc(NVAR), Aout(NREACT)
+    REAL(dp)               :: Vloc(NVAR),     Aout(NREACT)
 #ifdef MODEL_GEOS
     REAL(f4)               :: NOxTau, NOxConc, Vdotout(NVAR)
+    REAL(dp)               :: localC(NSPEC)
```

Comment:
localC is needed for MODEL_GEOS and MODEL_WRF, in order to restore the prior concentrations.